Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -283,3 +296,31 @@ const std::string GameListItem::GetWiiFSPath() const
 	return ret;
 }
 
+bool GameListItem::IsElfOrDol() const
+{
+	const std::string name = GetName();
+	const size_t pos = name.rfind('.');
+
+	if (pos != std::string::npos)
+	{
+		std::string ext = name.substr(pos);
+		std::transform(ext.begin(), ext.end(), ext.begin(), ::tolower);
+
+		return ext == ".elf" ||
+		       ext == ".dol";
+	}
+	return false;
+}
+
+bool GameListItem::SupportsCompression() const
+{
+	if (GetPlatform() == DiscIO::IVolume::GAMECUBE_DISC ||
+		GetPlatform() == DiscIO::IVolume::WII_DISC)
```

Comment:
The second tab here needs to be replaced with spaces. Also, you can return the condition directly, removing the if statement.